_posts/2016-12-07-GET_Deprecated_on_Some_Endpoints.md

```diff
@@ -21,7 +21,7 @@ Your app may need to change in order to continue reading data from our API.
 
 ## Deprecated Capabilities
 
-In the endpoints enumerated above, we are deprecating the broad functionality of the collection GET method i.e. {{ site.github.url }}/docs/v71_Sleep#collection.
```

Comment:
This post has the only observable difference by using relative URLs, but could either not be considered an issue, or could use `{{ site.github.url | replace: 'http:','https:' }}`